src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,201 @@
+    var builder = [];
+
+    // Show comma before primary and not secondary attributes
+    const status = key in config ? config[key].primary : true;
+    const formatter =
+        key in config && 'formatter' in config[key]
+            ? config[key].formatter
+            : t => {
+                  return `${t}`;
+              };
+    const prefix = status !== true ? '' : ', ';
+
+    builder.push(prefix);
+    builder.push(formatter(value));
+    return builder.join('');
+};
+
+/**
+ *  Gets a color for the given key, value, and config. 
+    If there a specified color for the key and value, return it.
+    If there is no such value, then get the color only specified for the key. 
+    Otherwise, return a default color. 
+ * @param {string} value 
+ * @param {string} key 
+ * @param {object} config 
+ */
+export const colorGetter = (value, key, config) => {
+    // Style based on attr-id only and not attr-value
+    const color = 'black';
+    const NO_VALUE = '_';
+    const existsKey = key in config && 'values' in config[key];
+
+    return existsKey && value in config[key].values
+        ? config[key].values[value]
+        : existsKey && NO_VALUE in config[key].values
+        ? config[key].values[NO_VALUE]
+        : color;
+};
+
+// Colors taken from clinicalAttributes.scss
+export const colors = {
+    sample_color_primary: 'black',
+    sample_color_recurrence: 'orange',
+    sample_color_metastasis: 'red',
+    sample_color_cfdna: 'blue',
+    sample_color_xenograft: 'pink',
+};
+
+/**
+ * Object mapping of keys to objects indicating
+ * whether an attribute is primary or secondary,
+ * the formatter to display the inputted value t,
+ * and font coloring based on the key and value.
+ */
+export const config = {
+    OS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t} months)`;
+        },
+    },
+    DFS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t} months)`;
+        },
+    },
+    CANCER_TYPE_DETAILED: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    PATIENT_DISPLAY_NAME: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    DERIVED_SAMPLE_LOCATION: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    SAMPLE_DISPLAY_NAME: {
+        primary: false,
+        formatter: t => {
+            return `\xa0(${t})`;
+        },
+    },
+    SERUM_PSA: {
+        primary: true,
+        formatter: t => {
+            return `, Serum PSA: ${t}`;
+        },
+    },
+    ERG_FUSION_ACGH: {
+        primary: true,
+        formatter: t => {
+            return `, ERG-fusion aCGH: ${t}`;
+        },
+    },
+    TMPRSS2_ERG_FUSION_STATUS: {
+        primary: true,
+        formatter: t => {
+            return `, TMPRSS2-ERG Fusion: ${t}`;
```

Comment:
again the comma doesn't need to be embedded in the data (and duplicated).  it should go in the getSpanElementsFromCleanData function down below